react/features/recording/components/LiveStream/web/StreamKeyPicker.js

```diff
@@ -111,15 +111,9 @@ class StreamKeyPicker extends PureComponent<Props, State> {
         const dropdownItems
             = broadcasts.map(broadcast => (
                 <DropdownItem
+                    data-streamid = { broadcast.boundStreamID }
```

Comment:
This was done to get rid of `eslint-disable-next-line react/jsx-no-bind`. Before this change it was creating new arrow functions inline on each render which can cause unnecessary rerenders because react has no way to detect that the new arrow function passed to the event handler prop is basically doing the same as the old one. We added a static event handler function that now gets the streamId from the element that is given in the event args of the keypress or click event.